src/controller/data_model/controller-clusters.matter

```diff
@@ -2587,6 +2587,7 @@ cluster BridgedDeviceBasicInformation = 57 {
 
   request struct KeepActiveRequest {
     int32u stayActiveDuration = 0;
+    int32u timeoutMs = 1;
```

Comment:
Why use 1 as the default value? this number is not within the valid range( 30000 to 3600000)

Reply:
1 is the field id, not the value.